zerver/lib/actions.py

```diff
@@ -4366,10 +4366,39 @@ def do_send_create_user_group_event(user_group: UserGroup, members: List[UserPro
                  )
     send_event(event, active_user_ids(user_group.realm_id))
 
-def check_add_user_group(realm: Realm, name: Text, initial_members: List[UserProfile],
+def send_user_group_update_notification(acting_user: UserProfile, group_name: Text,
+                                        members: List[UserProfile], action: Text) -> None:
+    if action == "add_members":
+        message_template = """Hi there! We thought you'd like to know that {} ({}) \
```

Comment:
You've been added to the user group {}. You can [view the other members of the group], or use `@*{}*` to mention everyone in the group.

Reply:
We don't have a mechanism to link to the members of the group beyond just sending them to the user groups management page.

Reply:
I think linking to the user groups management page is good.